### PR TITLE
feat: Subway Status 2.0 - LCD Frontend

### PR DESCRIPTION
**Pull request by jzimbel-mbta**
**Asana task**: [[Subway Status 2.0] LCD frontend changes](https://app.asana.com/0/1185117109217413/1203775859825085/f)

Most of the files in the diff are just newly-added pill SVGs! The meat of the changes is in subway_status_v2.tsx and subway_status_v2.scss.

- [ ] Tests added?

### CODE REVIEW

File: assets/src/components/v2/lcd/subway_status_v2.tsx [hunk 1]
```diff
@@ -0,0 +1,466 @@
...
+      abbrev = true;
+      break;
+    case FittingStep.PerAlertEffect:
+      abbrev = true;
+      switch (firstWord(status)) {
+        case "Delays":
+          truncateStatus = true;
+          break;
+        case "Suspension":
+        case "Bypassing":
```
[the remark this answers is not in the document]
Comment (cmaddox5) [reply assets/src/components/v2/lcd/subway_status_v2.tsx:208]: (We chatted about this in Slack so this may be fixed by the time this PR is submitted.)

**Review by cmaddox5 — COMMENTED** [review]
Awesome work! Love all the documentation. I did find an issue with suspensions and station closures showing the URL instead of the location, but I know you are aware of it already.

File: assets/css/v2/lcd_common_styles/subway_status_v2.scss [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+  .subway-status_status--contracted.subway-status_status--two-alerts:not(.subway-status_status--has-second-pill) {
+    height: 228px;
+  }
+
+  // height not needed for 2 alerts + 2 pills because this can only happen for the GL status,
+  // which always appears last and has nothing below it (not even the last
+  // rule, since the 2nd pill means we don't show it--see `shouldShowLastRule` JS function)
+  //
+  // &--contracted.subway-status_status--two-alerts.subway-status_status--has-second-pill {}
```
[the remark this answers is not in the document]
Comment (jzimbel-mbta) [reply assets/css/v2/lcd_common_styles/subway_status_v2.scss:57]: It was intended as a sort of—"You might think styles for this selector is needed, but it's not and here's why"

But maybe that's unnecessary... If you'd still like it removed I'd be happy to. I adjusted the comment's positioning slightly to try and make it clearer.

File: assets/src/components/v2/lcd/subway_status_v2.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (jzimbel-mbta) [reply assets/src/components/v2/lcd/subway_status_v2.tsx:209]: Yes, fixed now! I previously had an incorrect style for the intermediate layout while sizing is still happening, which made `useTextResizer` always choose the smallest size.

File: assets/css/v2/lcd_common_styles/subway_status_v2.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (cmaddox5) [reply assets/css/v2/lcd_common_styles/subway_status_v2.scss:58]: O ok, I misunderstood why it was commented out. I think it's fine to leave in, especially with the comment below it.

**Review by cmaddox5 — APPROVED** [review]
🎸

**Pull request merged**